apps/user_profile/models.py

```diff
@@ -91,6 +92,59 @@ class Child(models.Model):
         blank=False,
         verbose_name='Стать'
     )
+
+    city = models.CharField(
```

Comment:
this is should be FK to https://github.com/aodarc/final_project/blob/0481c12b365f32a14124b9d4ff270ca83cdbc70c/apps/location/models.py#L15